taskcluster/ci/nimbledroid/kind.yml

```diff
@@ -12,30 +12,37 @@ kind-dependencies:
     - signing
 
 
+job-defaults:
+    description: Upload APKs to Nimbledroid for performance measurement and tracking
+    run:
```

Comment:
Bug: Add 

```yaml
            using: run-commands
            use-caches: false
```

to this section. That's what this cryptic error is for. I'm sorry the message is really bad.

What these 2 lines mean: 
* `using: run-commands` => It tells taskgraph you're able to use `commands`. Without it, taskgraph is lost and doesn't know what `run` should look like.
* `use-caches: false` => it tells taskgraph you don't want to use existing cache. You have to be explicit about it, if I remember correctly.